### PR TITLE
Replaced eregi with preg_match

### PR DESCRIPTION
**Pull request by trick77**

### CODE REVIEW

File: includes/common.php [hunk 1]
```diff
@@ -699,8 +699,8 @@ function get_smokeping_files($device) {
         if ($handle = opendir($smokeping_dir)) {
             while (false !== ($file = readdir($handle))) {
                 if ($file != '.' && $file != '..') {
-                    if (eregi('.rrd', $file)) {
-                        if (eregi('~', $file)) {
+                    if (preg_match('/.rrd/i', $file)) {
```
Comment (laf) [inline includes/common.php]: Does this need to be \. to escape the full stop?

**Comment by trick77** [comment]
Do you mean like
if (preg_match('/\.rrd/i', $file)) {

**Comment by laf** [comment]
Yeah

**Comment by trick77** [comment]
You're right. Since there are quite a few functions that don't work properly in 16.04 I'm closing this PR.

File: includes/common.php [hunk 2]
```diff
@@ -699,8 +699,8 @@ function get_smokeping_files($device) {
...
+                        if (preg_match('/~/i', $file)) {
```
Comment (murrant) [inline includes/common.php]: It looks like you only need `strpos($file, '~') !== false` here

**Pull request closed without merging**